src/@dvcorg/gatsby-theme-iterative/redirects.js

```diff
@@ -0,0 +1 @@
+module.exports = require('../../../redirects-list.json')
```

Comment:
Must be exported from here to shadow, the file contents could also live here (and be my preference) but I figured I'd do what I can to minimize the amount of unnecessary changes tossed onto this already massive pile.